### ISSUE TITLE
Splashscreen logo not showing after first time open after installation in many android devices (After upgarding to cordova 11)

### ISSUE THREAD

**Issue by champikamendis** [apache/cordova-android#1600]
## Problem
After installing the application first run, the splashscreen logo is missing and the very next time and onwards it shows well.
This happens to my real devices. In Pixel 7 Pro, OnePlus 8T both(Android 13) this happened.
### What is expected to happen?
Expected to show the splash screen logo at the first run of the application after installation.

### What does actually happen?
In the first run after installation splash screen logo is missing but the background loading fine.
![splashscreenlogomissing](https://user-images.githubusercontent.com/37321146/233009682-417b1777-9fc6-4276-b729-e8a004974ff5.png)

## Information
I have tried following scenarios.

- Added png as the icon of the splash screen
- Added xml as the icon (Tried adaptive icon concept)
- In the AndroidWindowSplashScreenAnimatedIcon value attribute I placed null and checked(then default cordova logo should be loaded. But same issue)

### Command or Code
`<preference name="AndroidWindowSplashScreenAnimatedIcon" value="res/splash/layout/activity_main.xml" />
  <preference name="AndroidWindowSplashScreenBackground" value="#1d396b" />
 <preference name="ShowSplashScreenSpinner" value="true" />
 <preference name="AutoHideSplashScreen" value="true" />`

### Environment, Platform, Device
Platform: Android
Devices: OnePlus 8T
                Pixel 7 Pro

### Version information
Cordova Android: 11

## Checklist

- [x ] I searched for existing GitHub issues
- [x ] I updated all Cordova tooling to most recent version
- [x ] I included all the necessary information above

**Comment by breautek** [comment]
Please share the contents of your `<cordova-project>/platforms/android/app/src/main/res/values/themes.xml` file.

**Comment by champikamendis** [comment]
Here is the requested themes.xml file inside the relevant platform
`<?xml version='1.0' encoding='utf-8'?>

<resources>

    <style name="Theme.App.SplashScreen" parent="Theme.SplashScreen.IconBackground">

        <item name="windowSplashScreenBackground">@color/cdv_splashscreen_background</item>

        <item name="windowSplashScreenAnimatedIcon">@drawable/ic_cdv_splashscreen</item>

        <item name="windowSplashScreenAnimationDuration">200</item>

        <item name="postSplashScreenTheme">@style/Theme.AppCompat.NoActionBar</item>

    </style>

</resources>
`

**Comment by breautek** [comment]
Thank you.

The themes file looks fine...  so the next thing I'd like to check/inspect is your actual image resource.

Can you share your `res/splash/layout/activity_main.xml` file?

**Comment by champikamendis** [comment]
Thanks for your quick response! Here is my res file
`
<vector android:height="24dp" android:tint="#A03838"
    android:viewportHeight="24" android:viewportWidth="24"
    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
    <path android:fillColor="@android:color/white" android:pathData="M13,24c-3.26,0 -6.19,-1.99 -7.4,-5.02l-3.03,-7.61C2.26,10.58 3,9.79 3.81,10.05l0.79,0.26c0.56,0.18 1.02,0.61 1.24,1.16L7.25,15H8V3.25C8,2.56 8.56,2 9.25,2s1.25,0.56 1.25,1.25V12h1V1.25C11.5,0.56 12.06,0 12.75,0S14,0.56 14,1.25V12h1V2.75c0,-0.69 0.56,-1.25 1.25,-1.25c0.69,0 1.25,0.56 1.25,1.25V12h1V5.75c0,-0.69 0.56,-1.25 1.25,-1.25S21,5.06 21,5.75V16C21,20.42 17.42,24 13,24z"/>
</vector>
`

**Comment by champikamendis** [comment]
Any update?

**Comment by dylanfpaul** [comment]
Having the same issue. Tried linking a generated xml file as per a guide, as well as a png. I always get the default cordova splashscreen even on a real device.

**Comment by breautek** [comment]
I think it's related to https://issuetracker.google.com/issues/244063660

I don't think this is a bug with Cordova.

Does this issue only present itself when launching from Android Studio and/or CLI?

**Comment by jcesarmobile** [comment]
yes, this is an Android bug, nothing we can do on our end, Google should fix it
